src/styles/_base.scss

```diff
@@ -127,6 +127,7 @@ ul {
   display: grid;
   flex: 1 1 auto;
   grid-template-rows: 100%;
+  min-height: calc(100vh - var(--page-header-height) - 400px);
```

Comment:
When there was not enough content on the page (e.g. on the data privacy category page) or if the user was zoomed out by a lot, there was extra space underneath the footer.